PatientRegistrationApp/src/components/HouseholdReviewSubmit.vue

```diff
@@ -38,6 +38,11 @@
 			<v-col cols="12">
 				<v-card class="d-flex flex-wrap" flat>
 					<v-card class="pa-2" flat min-width=33%>
+						<v-img
+							max-height="100"
```

Comment:
Not sure if this is where the formatting of the photo for display on the final review page occurs, but the photo seems too small for use here. We should also consider that this photo will be saved into our db and displayed in the POD SW where we alott a pretty big space for it. 

It also might be of worth to add a comment or some check so you don't get off centered photos like below... 

![image](https://user-images.githubusercontent.com/67079030/100696486-42c14380-3359-11eb-88cd-b2f807434f87.png)